CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Fixed `Color.__get___` AttributeError.
 
+* Fixed  `RhinoSurface.curvature_at` not returning a Vector, but a Rhino SurfaceCurvature class object 
```

Comment:
never really understand why people do this :)
could you move to line 23 and leave 24 blank?

Reply:
I thought it was to separate different contributors, I am removing it!